meta-oe/recipes-devtools/android-tools/android-tools_5.1.1.r37.bb

```diff
@@ -92,7 +92,7 @@ do_compile() {
 
     # Setting both variables below causing our makefiles to not work with
     # implicit make rules
-    unset CFLAGS
+    CFLAGS="--file-prefix-map=${S}=/usr/src/debug/${PN}/${EXTENDPE}${PV}-${PR}"
```

Comment:
This should be `-ffile-prefix-map`

Reply:
Changed.
Interestingly `--file-prefix-map` also works, but `-ffile-prefix-map` is the documented option.